pkgs/by-name/fl/flatpak/unset-env-vars.patch

```diff
@@ -1,12 +1,12 @@
-diff --git a/common/flatpak-run.c b/common/flatpak-run.c
 index e3f031d4..ed131c0b 100644
 --- a/common/flatpak-run.c
 +++ b/common/flatpak-run.c
-@@ -571,6 +571,7 @@ static const ExportData default_exports[] = {
+@@ -571,6 +571,8 @@ static const ExportData default_exports[] = {
    {"XKB_CONFIG_ROOT", NULL},
    {"GIO_EXTRA_MODULES", NULL},
    {"GDK_BACKEND", NULL},
 +  {"GDK_PIXBUF_MODULE_FILE", NULL},
++  {"TZDIR", NULL},
```

Comment:
We may want to pick up https://github.com/flatpak/flatpak/pull/5942 once merged.

Reply:
It is my understanding it will go into the 1.15.X releases until 1.16 is released, which is when both this and the patches can be removed. Until then I think this works fine. It fixes the problem